### PR TITLE
BREAKING CHANGE(web-twig): Set isExpandedOnMobile in ModalDialog to false by default #DS-1201

### PR DESCRIPTION
**Pull request by crishpeen**

### CODE REVIEW

**Review by literat — APPROVED** [review]
LGTM 👍

File: packages/web-react/src/components/Modal/README.md [hunk 1]
```diff
@@ -163,16 +161,16 @@ You can use the `maxHeightFromTabletUp` option to override the max height on tab
...
+| ----------------------------- | --------------------- | --------- | -------- | ---------------------------------------------------------------------------------------------------------------------------------------- |
+| `children`                    | `ReactNode`           | —         | ✕        | Children node                                                                                                                            |
+| `elementType`                 | [`article` \| `form`] | `article` | ✕        | ModalDialog element type                                                                                                                 |
+| `isDockedOnMobile`            | `bool`                | `false`   | ✕        | [REQUIRES FEATURE FLAG](#feature-flag-uniform-appearance-on-all-breakpoints): Dock the ModalDialog to the bottom of the screen on mobile |
+| `isExpandedOnMobile`          | `bool`                | `false`   | ✕        | ModalDialog shrinks to fit the height of its content.                                                                                    |
```
Comment (adamkudrna) [inline packages/web-react/src/components/Modal/README.md:169]: Please fix the description, now it says the opposite 🙂.

```suggestion
| `isExpandedOnMobile`          | `bool`                | `false`   | ✕        | If true, ModalDialog expands to fit the viewport                                                                                    |
```

(With no full stops anywhere, I believe?)

File: packages/web-react/src/components/Modal/README.md [hunk 2]
```diff
@@ -25,6 +28,7 @@ Modal is a composition of several subcomponents:
   - [Opening the Modal](#opening-the-modal)
   - [Scrolling Long Content](#scrolling-long-content)
     - [Scrolling with ScrollView](#scrolling-with-scrollview)
+    - [Disable Scrolling Inside ModalDialog](#disable-scrolling-inside-modaldialog)
   - [Stacking Modals](#stacking-modals)
   - [Full Example](#full-example)
```
Comment (adamkudrna) [inline packages/web-react/src/components/Modal/README.md:33]: Thanks for completing the TOC!

**Review by adamkudrna — APPROVED** [review]

File: packages/web-react/src/components/Modal/README.md (hunk 1 above)
Comment (adamkudrna) [reply packages/web-react/src/components/Modal/README.md:169]: … on mobile, of course.

**Pull request merged**